backend/app/api/game.py

```diff
@@ -157,10 +157,25 @@ def get_tile():
 
     return tile
 
-
+"""function to filter chat messages"""
 def filter_message(data: Dict[str, str]) -> Dict[str, str]:
-    """function to filter chat messages"""
 
+    bannedWords = ["shit","fuck","crap","bitch"]
+    censoredSentence = []
+    " ".join(censoredSentence)
```

Comment:
you need to move the join statement to the end of the function because there's nothing in `censoredSentence` at the time you do the join, so this will just produce an empty string. Furthermore, `str.join()` returns a [str](https://docs.python.org/3/library/stdtypes.html#str.join) but you haven't assigned it to a variable, so this line wouldn't do anything anyway.